rpm/openssl.spec

```diff
@@ -1,483 +1,246 @@
+BuildRequires:  perl(File::Temp)
+BuildRequires:  perl(FindBin)
+BuildRequires:  perl(IPC::Cmd)
+BuildRequires:  perl(IPC::Cmd)
+BuildRequires:  perl(Math::BigInt)
+BuildRequires:  perl(Module::Load::Conditional)
+BuildRequires:  perl(Pod::Html)
+BuildRequires:  perl(Test::Harness)
+BuildRequires:  perl(Test::More)
+BuildRequires:  perl(Time::HiRes)
+BuildRequires:  perl(bigint)
+BuildRequires:  perl(lib)
+BuildRequires:  pkgconfig(zlib)
+Requires:       openssl
+Requires:       openssl-libs = %{version}-%{release}
+Provides:       ssl
+# Requires:       crypto-policies
 
 %description
-The OpenSSL toolkit provides support for secure communications between
-machines. OpenSSL includes a certificate management tool and shared
-libraries which provide various cryptographic algorithms and
-protocols.
+OpenSSL is a software library to be used in applications that need to
+secure communications over computer networks against eavesdropping or
+need to ascertain the identity of the party at the other end.
+OpenSSL contains an implementation of the SSL and TLS protocols.
 
 %package libs
-Summary: A general purpose cryptography library with TLS implementation
-Requires: ca-certificates >= 2008-5
-# Needed obsoletes due to the base/lib subpackage split
-Obsoletes: openssl < 1.0.1b
+Summary:        Secure Sockets and Transport Layer Security
+Requires:       ca-certificates
+Conflicts:      %{name} < %{version}-%{release}
+# Requires:       crypto-policies
 
 %description libs
-OpenSSL is a toolkit for supporting cryptography. The openssl-libs
-package contains the libraries that are used by various applications which
-support cryptographic algorithms and protocols.
+OpenSSL is a software library to be used in applications that need to
+secure communications over computer networks against eavesdropping or
+need to ascertain the identity of the party at the other end.
+OpenSSL contains an implementation of the SSL and TLS protocols.
 
 %package devel
-Summary: Files for development of applications which will use OpenSSL
-Requires: %{name}-libs%{?_isa} = %{version}-%{release}
-Requires: zlib-devel%{?_isa}
-Requires: pkgconfig
+Summary:        Development files for OpenSSL
+Requires:       openssl-libs = %{version}
+Requires:       pkgconfig(zlib)
+Recommends:     %{name} = %{version}
+Obsoletes:      openssl-devel < %{version}
 
 %description devel
-OpenSSL is a toolkit for supporting cryptography. The openssl-devel
-package contains include files needed to develop applications which
-support various cryptographic algorithms and protocols.
-
-%package static
-Summary:  Libraries for static linking of applications which will use OpenSSL
-Requires: %{name}-devel%{?_isa} = %{version}-%{release}
-
-%description static
-OpenSSL is a toolkit for supporting cryptography. The openssl-static
-package contains static libraries needed for static linking of
-applications which support various cryptographic algorithms and
-protocols.
-
-%package perl
```

Comment:
Please keep the perl subpackage. I don't want it pulled in unless really really needed.